go/vt/vttest/environment.go

```diff
@@ -155,6 +163,22 @@ func (env *LocalTestEnv) MySQLManager(mycnf []string, snapshot string) (MySQLMan
 	}, nil
 }
 
+func isRunningMariaDB() bool {
```

Comment:
It might be good to add a `IsMariaDB()` API to `mysqlctl` where it will can be implemented just as `return capabilities.isMariaDB()`

Reply:
This is already there in the `mysql` package's flavor interface:
```
// IsMariaDB returns true iff the other side of the client connection
// is identified as MariaDB. Most applications should not care, but
// this is useful in tests.
func (c *Conn) IsMariaDB() bool {
	switch c.flavor.(type) {
	case mariadbFlavor101, mariadbFlavor102:
		return true
	}
	return false
}
```